bodhi-client/pyproject.toml

```diff
@@ -44,6 +44,7 @@ click = ">=7.1.2"
 koji = "^1.27.1"
 authlib = ">=0.15.4"
 munch = "^2.5.0"
+requests-kerberos = ">=0.12"
```

Comment:
If you change the `pyproject.toml` file, you need to regenerate the lock file with `poetry lock`.